Yashgabani845/hiring-portal
# fix: [FEATURE] Added Contact Us Page

### Added Contact Us Page

**Issue Reference:**
This PR addresses issue #78 

**Description:**
This PR introduces a new "Contact Us" page to the website. The page includes a form for users that'll used to submit inquiries, along with the contact information.

**How to Test**
1) Navigate to the "Contact Us" page through the links in both the navbar and footer.
2) Verify that the layout is responsive on different screen sizes.
3) Check that the links in the navbar and footer correctly direct to the new page.

**Screenshots**

![image](https://github.com/user-attachments/assets/3b2ae132-5f2d-433b-9afa-4e35061d1efc)

![image](https://github.com/user-attachments/assets/08ccbec6-0429-4f75-8584-443f20c190d4)

**Type of change:**

**Checklist:**
- [x] I have tested my changes and verified that they work

**Additional Information:**
Fixes issue #78
